### PR TITLE
Also deserialize PURLs from the model

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: cli/src/main/kotlin/commands/ReporterCommand.kt [hunk 1]
```diff
@@ -276,7 +276,8 @@ class ReporterCommand : CliktCommand(
             val durationInSeconds = timedValue.duration.inSeconds
 
             timedValue.value.onSuccess { files ->
-                println("Successfully created the '$name' report at $files in ${durationInSeconds}s.")
+                val fileList = files.joinToString { "'$it'" }
+                println("Successfully created '$name' report(s) at $fileList in ${durationInSeconds}s.")
```
Comment (mnonnenmacher) [inline cli/src/main/kotlin/commands/ReporterCommand.kt:280]: Commit doesn't belong to this PR.

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

**Review by mnonnenmacher — APPROVED** [review]

**Comment by sschuberth** [comment]
Merging despite the unrelated test failures, which will be fixed by https://github.com/oss-review-toolkit/ort/pull/3986.
